pkg/resource/prunercronjob.go

```diff
@@ -130,6 +140,15 @@ func (gcj *generatorPrunerCronJob) expected() (runtime.Object, error) {
 	return cj, nil
 }
 
+func (gcj *generatorPrunerCronJob) getPruneRegistry(cr *imageregistryapiv1.Config) bool {
```

Comment:
nit: this function could be converted into `return cr.Spec.ManagementState == operatorapi.Managed`. the function name seems misguiding as well as it does not return a "PruneRegistry" :-)

Reply:
The function handles three cases, Managed, Removed, and UnManaged, so the == would not handle all three.
I feel like the getPruneRegistry is inline with the rest of the get(functionNames) here.  Feel free to suggest a new name.